@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -457,26 +458,49 @@ export function applyTechniqueToMaterial(
     level?: number,
     skipExtraProps?: string[]
 ) {
-    Object.getOwnPropertyNames(technique).forEach(propertyName => {
+    const commonProps = Object.getOwnPropertyNames(technique).filter(propertyName => {
```

Comment:
> `commonProps` -> `genericProps`

These props  `generic` as they're excercising generic/normal flow in contrast to filtered ones which are ignored or ones that require specific flow.